authors/apps/bookmark/models.py

```diff
@@ -0,0 +1,28 @@
+"""
+    Models for Bookmark app
+"""
+from django.db import models
+
+# local imports
+from authors.apps.articles.models import Articles
+from authors.apps.profiles.models import Profile
+
+
+# Create your models here.
+class Bookmark(models.Model):
+    """
+        Define the model for Bookmark
+    """
+    # Bookmark can only be either 1 or -1
+    BOOKMARK_CHOICES = (
```

Comment:
@EmmanuelBeja , this would be best configured as a Boolean (`True/False`), to indicate whether or not a user has bookmarked a certain article. The db would then store references to the articles a user has bookmarked. 
As it currently looks, it looks like a user can unbookmark an article, and the db would store the articles a user has unbookmarked. I don't think this is important to store.

Reply:
mmmh. noted.